pkg/utils/misc.go

```diff
@@ -0,0 +1,13 @@
+package utils
```

Comment:
If we add this a more appropriate 'pkg' name would be "stringutil".  Code under pkg shouldn't have knowledge of snap and as such would be available for general reuse therefore the package name should be as self describing as possible.  

Outside of a test this function is only called once in wmap and with only a single function, in the pkg, it's not very useful to anyone else IMO.  I would prefer that we not break it out into a pkg at this time and we move the functionality back into wmap even if we have to duplicate the code in metric_test.go (since it's the other place it's being called).

Lastly, here is an alternate implementation that avoids the range and fmt.Sprintf.
```
func GetFirstChar(s string) string {
	if len(s) > 0 {
		return s[:1]
	}
	return ""
}
```